src/Accordion/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { string, bool, element, oneOfType, func } from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from 'watson-react-components';
+
+Accordion.propTypes = {
+  show: bool,
+  content: element.isRequired,
```

Comment:
Depending on what `content` could be, it might be better to use `node` here since that will allow you to pass in an array of elements, a single element, simple strings, etc. If you're always expecting a single react element, then ignore this.